rollup.config.js

```diff
@@ -24,24 +24,35 @@ const LEGAL_TEXT = `/*
 * Attribution and license information can be found at https://github.com/MyPureCloud/client-app-sdk/blob/master/README.md
 */`;
 
+// Packages to exclude from esm/cjs bundles
+const EXTERNAL_DEPS = [
+    'query-string',
+    /@babel\/runtime/ // Need to use a regex here: https://github.com/rollup/plugins/issues/475#issuecomment-652416290
+];
+
+// Need to be able to configure whether or not
+// babel pulls in the esm versions of the runtime helpers
+const babelPlugin = (useESModules = false) => babel({
+    babelHelpers: 'runtime',
+    exclude: /node_modules/,
+    presets: ['@babel/env'],
+    plugins: [['@babel/transform-runtime', { useESModules }]]
```

Comment:
Opted to just leave the babel config in here instead of moving it into a separate `.babelrc` file as it would really only add complexity due to the `useESModules` flag.